_posts/2021-01-04-proxmox-server.md

```diff
@@ -0,0 +1,249 @@
+    lxc.mount.entry: /dev/nvidia-uvm dev/nvidia-uvm none bind,optional,create=file
+    lxc.mount.entry: /dev/nvidia-modeset dev/nvidia-modeset none bind,optional,create=file
+    lxc.mount.entry: /dev/nvidia-uvm-tools dev/nvidia-uvm-tools none bind,optional,create=file
+    ```
+
+    Reboot the container for the settings to take effect. After the reboot check to see if the configurations worked with the following commands and outputs:
+    ``` bash
+    # nvidia-smi
+    Tue Jan  5 02:57:07 2021
+    +-----------------------------------------------------------------------------+
+    | NVIDIA-SMI 450.80.02    Driver Version: 450.80.02    CUDA Version: 11.0     |
+    |-------------------------------+----------------------+----------------------+
+    | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+    | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+    |                               |                      |               MIG M. |
+    |===============================+======================+======================|
+    |   0  GeForce GTX 107...  Off  | 00000000:01:00.0  On |                  N/A |
+    |  0%   45C    P8     8W / 180W |      1MiB /  8116MiB |      0%      Default |
+    |                               |                      |                  N/A |
+    +-------------------------------+----------------------+----------------------+
+
+    +-----------------------------------------------------------------------------+
+    | Processes:                                                                  |
+    |  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+    |        ID   ID                                                   Usage      |
+    |=============================================================================|
+    |  No running processes found                                                 |
+    +-----------------------------------------------------------------------------+
+    ```
+    and
+    ``` bash
+    # ls /dev/nvidia* -l
+    -rw-r — r — 1 root root 0 16.01.2017 20:11 /dev/nvidia-modeset
+    crw-rw-rw- 1 nobody nobody 243, 0 16.01.2017 20:05 /dev/nvidia-uvm
+    -rw-r — r — 1 root root 0 16.01.2017 20:11 /dev/nvidia-uvm-tools
+    crw-rw-rw- 1 nobody nobody 195, 0 16.01.2017 20:05 /dev/nvidia0
+    crw-rw-rw- 1 nobody nobody 195, 255 16.01.2017 20:05 /dev/nvidiactl
+    ```
+
+### Creating Snapshots and Templates
+
+    After all this hard work, you should save your progress on the container by creating a snapshot. If you chose ZFS as the base file system, this would be an option in the web interface under the specific container's menu. 
+
+    You could also make the new GPU enabled container a template as a basis for other containers. The option of making that cointainer a template can be found via right-clicking on the name of the container. 
```

Comment:
formatting is off in markdown. you may have not closed code formatting properly in the section before this one

Reply:
fixed. spaces in the new sentences means quote in MD.